packages/server-renderer/src/helpers/ssrRenderSlot.ts

```diff
@@ -60,7 +62,7 @@ export function ssrRenderSlotInner(
       parentComponent,
       slotScopeId ? ' ' + slotScopeId : '',
     )
-    if (isArray(ret)) {
+    if (isArray(ret) && ensureValidVNode(ret)) {
```

Comment:
I think the proper fix should be:

```ts
      // normal slot
      const validSlotContent = ensureValidVNode(ret)
      if (validSlotContent) {
        renderVNodeChildren(push, validSlotContent, parentComponent, slotScopeId)
      } else if (fallbackRenderFn) {
        fallbackRenderFn()
      }
```
Do not fall back to the processing logic of the ssr slot.

Reply:
thanks for your replay. I have corrected it.